src/app/components/contentTile/contentTile.component.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import './contentTile.component.scss';
+import { ContentItem, ContentLabel } from '../../modules/content/content.state';
+
+export interface ContentTileProps {
+    item: ContentItem;
+    label: ContentLabel;
+    specifiedLength: number;
+}
+
+export function ContentTile({ item, label, specifiedLength }: ContentTileProps) {
+    const summaryLength = item.summary.length;
+    const usedLength = specifiedLength > summaryLength ? summaryLength-3 : specifiedLength;
+
+    return (
+        <section
```

Comment:
do not return such complicated template, extract to separate function

Reply:
Created utils functions, as mentioned in another comment.